### PR TITLE
batches: update UI changelog and constants for release

### PR DESCRIPTION
**Pull request by LawnGnome**
Wording bikeshedding welcome! I've written this assuming #20361 lands before the release is cut; we'll just excise that line if it doesn't look like it'll make it.

### CODE REVIEW

File: client/web/src/enterprise/batches/list/BatchChangesListIntro.tsx [hunk 1]
```diff
@@ -60,31 +60,25 @@ const BatchChangesChangelogAlert: React.FunctionComponent = () => (
                     </li>
                 </ul>
                 <ul className="text-muted mb-0 pl-3">
-                    <li>
-                        Site admins can now configure a global service account to be used when creating and updating
-                        changesets on code hosts and the user has no credential configured.
-                    </li>
+                    {/* TODO: link to documentation if we have it; remove if this doesn't make it before branch cut. */}
```
Comment (eseliger) [inline client/web/src/enterprise/batches/list/BatchChangesListIntro.tsx:63]: Merged, still need to create a link to documentation (and documentation itself)

Comment (eseliger) [reply client/web/src/enterprise/batches/list/BatchChangesListIntro.tsx:63]: https://docs.sourcegraph.com/batch_changes/how-tos/bulk_operations_on_changesets is in PR https://github.com/sourcegraph/sourcegraph/pull/21056

File: client/web/src/enterprise/batches/list/BatchChangesListIntro.tsx [hunk 2]
```diff
@@ -60,31 +60,25 @@ const BatchChangesChangelogAlert: React.FunctionComponent = () => (
...
+                    <li>Comments can be added to some or all changesets in a batch change.</li>
```
[the remark this answers is not in the document]
Comment (LawnGnome) [reply client/web/src/enterprise/batches/list/BatchChangesListIntro.tsx:64]: Sure, let's go with that. I was trying to imply the existence of bulk operations, but punchier is probably better here.

File: internal/src-cli/consts.go [hunk 3]
```diff
@@ -6,4 +6,4 @@ package srccli
 // as the suggested download with this instance.
 //
 // At the time of a Sourcegraph release, this is always the latest src-cli version.
-const MinimumVersion = "3.27.0"
+const MinimumVersion = "3.28.0"
```
Comment (eseliger) [inline internal/src-cli/consts.go:9]: Do we need to cut the 3.28 release _before_ merging this PR?

**Review by eseliger — APPROVED** [review]
LGTM! Do we also want to get rid of the `BatchChangesRenameAlert`?

File: internal/src-cli/consts.go (hunk 3 above)
Comment (LawnGnome) [reply internal/src-cli/consts.go:9]: I didn't think so, but now I look at the logic again for providing a download, we actually might. I'm going to re-tag and re-release 3.27.1 as 3.28.0, I think.

**Comment by LawnGnome** [comment]
> LGTM! Do we also want to get rid of the `BatchChangesRenameAlert`?

Sure, why not.

**Pull request merged**